sandbox-start.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euxo pipefail
+
+cd "$(dirname "${0}")"
+
+DAML_SRC=test/integration/src
+
+if [[ ! $(which daml) ]]; then
+    curl -sSL https://get.daml.com/ | sh
+    PATH="$PATH":"$HOME"/.daml/bin
+fi
+
+(cd "$DAML_SRC" && daml install 0.12.16 || true && daml build)
+
+PORT=$(comm -23 <(seq 49152 65535 | sort) <(ss -tan | awk '{print $4}' | cut -d':' -f2 | grep "[0-9]\{1,5\}" | sort -u) | shuf | head -n 1) || true
```

Comment:
This is where the magic happens 😈

Reply:
I looked for [my own answer on StackOverflow](https://unix.stackexchange.com/a/423052). :sweat_smile:

Reply:
But yeah, a comment is definitely due, thanks.

Reply:
Haha, classic. Current-Stefano is grateful for Past-Stefano's work :)